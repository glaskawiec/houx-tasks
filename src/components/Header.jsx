import React from 'react';
import styled from 'styled-components';

const Title = styled.h2`
  margin: 0;
  padding: 0.7em;
  text-align: left;
  color: yellow;
`;

const Header = () => (
  <header>
    <Title>
      Task Manager
    </Title>
  </header>
);

export default Header;
