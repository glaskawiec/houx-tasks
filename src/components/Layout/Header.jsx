import React from 'react';

export default function Header() {
    return (
        <header>
            <h2 className="apptitle">
                <span role="img" aria-label="logo">
                    âœ¨
                </span>
                houxTasks
            </h2>
        </header>
    )
}