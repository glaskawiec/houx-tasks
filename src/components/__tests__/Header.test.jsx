import { render } from '@testing-library/react';
import React from 'react';
import Header from '../Header';

describe('Header component unit tests', () => {
  test('Header should render properly', () => {
    const { container } = render(
      <Header />,
    );
    expect(container).toMatchSnapshot();
  });
});
