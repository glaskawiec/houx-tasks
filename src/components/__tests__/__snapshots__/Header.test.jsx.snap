// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component unit tests Header should render properly 1`] = `
<div>
  <header>
    <h2
      class="sc-AxjAm jytQxq"
    >
      Task Manager
    </h2>
  </header>
</div>
`;
